src/modules/facility-loan-transaction/dto/get-loan-transaction-params.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+
+export class GetLoanTransactionParamsDto {
+  @ValidatedStringApiProperty({ description: 'The UKEF identifier for the facility.', length: 10, pattern: /^\d{10}$/ })
+  facilityIdentifier: string;
```

Comment:
do we need these properties to be `readOnly`?

Reply:
Changed.